tests/mock_data.py

```diff
@@ -100,54 +100,109 @@
         "service-info"
     )
 }
-MOCK_FILES = [
-    {
-        "fileWrapper": {
-            "checksum": [
-                {
-                    "checksum": "checksum",
-                    "type": "sha1",
-                }
-            ],
-            "content": "content",
-            "url": "url",
-        },
-        "toolFile": {
-            "file_type": "TEST_FILE",
-            "path": "path",
-        }
+MOCK_TEST_FILE = {
+    "file_wrapper": {
+        "checksum": [
+            {
+                "checksum": "checksum",
+                "type": "sha1",
+            }
+        ],
+        "content": "content",
+        "url": "url",
     },
-    {
-        "fileWrapper": {
-            "checksum": [
-                {
-                    "checksum": "checksum",
-                    "type": "sha2",
-                }
-            ],
-            "content": "content",
-            "url": "url",
-        },
-        "toolFile": {
-            "file_type": "CONTAINERFILE",
-            "path": "path",
-        }
+    "tool_file": {
+        "file_type": "TEST_FILE",
+        "path": "path",
+    },
+    "type": "JSON"
+}
+MOCK_DESCRIPTOR_FILE = {
```

Comment:
You could simplify these by deep defining just one of them completely, then deep copying the others and replacing only what changes